Assets/MixedRealityToolkit/Attributes/MixedRealityControllerAttribute.cs

```diff
@@ -60,9 +61,18 @@ public class MixedRealityControllerAttribute : Attribute
         /// <summary>
         /// Convenience function for retrieving the attribute given a certain class type.
         /// </summary>
+        /// <remarks>
+        /// This function is only available in a UnityEditor context.
+        /// </remarks>
         public static MixedRealityControllerAttribute Find(Type type)
         {
+#if WINDOWS_UWP && !ENABLE_IL2CPP
+            // Type.GetCustomAttributes() doesn't exist on UWP .NET, so we have to indirectly use
+            // the TypeInfo instead.
+            return type.GetTypeInfo().GetCustomAttributes(typeof(MixedRealityControllerAttribute), true).FirstOrDefault() as MixedRealityControllerAttribute;
```

Comment:
What we generally did was implement a simple extension to Type that would look like this:

#if WINDOWS_UWP
public static TypeInfo AsTypeInfo(this Type type) => return type.GetTypeInfo();
#else
public static Type AsTypeInfo(this Type type) => return type;
#endif

This way, you could just us AsTypeInfo() through out and not worry

Reply:
Good eye, @andreiborodin! 

We do have some extensions (ReflectionExtensions.cs) and I believe this may be already provided.

Reply:
Done, thanks for the pointer!